examples/siro_sandbox/sandbox_app.py

```diff
@@ -437,6 +461,13 @@ def depth_to_rgb(obs):
 
         return post_sim_update_dict
 
+    def _try_next_state(self):
+        self._app_state_index += 1
+        if self._app_state_index >= len(self._app_states):
+            return
+        self._app_state = self._app_states[self._app_state_index]
+        self._app_state.on_environment_reset(self._episode_recorder_dict)
```

Comment:
This is weird and misleading. Since we haven't just reset the environment, we shouldn't be calling `on_environment_reset`.